mp/src/game/shared/momentum/fx_mom_shared.cpp

```diff
@@ -21,6 +21,9 @@
 
 #include "tier0/memdbgon.h"
 
+static MAKE_TOGGLE_CONVAR(mom_use_fixed_spread, "1", FCVAR_REPLICATED,
```

Comment:
Naming. Go with `mom_fixed_spread` instead.

Reply:
Should all be resolved in https://github.com/momentum-mod/game/commit/075e24f5df9193d6c44669dbb0bb68442d9d94b3